From 97dce95fdcb552246800dac12aab1234edae7f4f Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Fri, 17 Apr 2020 10:59:40 +0200
Subject: [PATCH 2/2] popover.rs: fix build with gtk-rs >= 0.7.0

---
 src/popover.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/popover.rs b/src/popover.rs
index ce1543b..03c6a8f 100644
--- a/src/popover.rs
+++ b/src/popover.rs
@@ -13,9 +13,9 @@ use ::resources;
 
 use gio::ActionMapExt;
 use gio::SettingsExt;
-use gio::SimpleActionExt;
 use glib::translate::FromGlibPtrNone;
 use glib::variant::ToVariant;
+use gtk::BuilderExtManual;
 use gtk::PopoverExt;
 use gtk::WidgetExt;
 use std::io::Write;
-- 
2.25.1
